### PR TITLE
Initial repository setup

### PR DESCRIPTION
**Pull request by amarthadan**
[BEC-245](https://api3dao.atlassian.net/browse/BEC-245)

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**/node_modules/**
```
Comment (Siegrift) [inline .eslintignore]: It's pretty disapointing that these ignore files do not have an option to extend gitignore (which seems a pretty reasonable default).

For that reason I've started to like putting a comment to keep them in sync. See: https://github.com/api3dao/manager-multisig/blob/main/.gitignore#L1

Also, you should probably put all of the gitignore stuff here.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: weekly
+      time: '11:00'
+    open-pull-requests-limit: 5
+    reviewers:
+      - amarthadan
```
Comment (Siegrift) [inline .github/dependabot.yml:10]: Feel free to add me as well.

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+**/dist/**
+.prettierignore
```
Comment (Siegrift) [inline .eslintignore]: Nit: \n

File: .prettierrc [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "bracketSpacing": true,
+  "printWidth": 120,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "useTabs": false,
+  "overrides": [
```
Comment (Siegrift) [inline .prettierrc:8]: I think you also want to add solidity support https://github.com/api3dao/airnode/pull/324/commits/e7c1bf033ac3e548b2bb8293a6abce5e8134a103

**Review by Siegrift — CHANGES_REQUESTED** [review]
Some minor changes, but otherwise LGTM.

File: .prettierrc (hunk 4 above)
Comment (amarthadan) [reply .prettierrc:8]: I'll do that once it's needed.

**Review by Siegrift — APPROVED** [review]
LGTM

**Pull request merged**